data_structures/stack/Infix_to_Postfix.C

```diff
@@ -0,0 +1,133 @@
+// C program to convert infix expression to postfix 
+#include <stdio.h> 
+#include <string.h> 
+#include <stdlib.h> 
+
+// Stack type 
+struct Stack 
+{ 
+	int top; 
+	unsigned capacity; 
+	int* array; 
+}; 
+
+// Stack Operations 
+struct Stack* createStack( unsigned capacity ) 
+{ 
+	struct Stack* stack = (struct Stack*) malloc(sizeof(struct Stack)); 
+
+	if (!stack) 
+		return NULL; 
+
+	stack->top = -1; 
+	stack->capacity = capacity; 
+
+	stack->array = (int*) malloc(stack->capacity * sizeof(int)); 
+
+	if (!stack->array) 
+		return NULL; 
```

Comment:
deallocate memory allocated to ``stack`` before returning to avoid memory leaks.